README.md

```diff
@@ -1,36 +1,47 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# AI Dashboard Generator
 
-## Getting Started
+AI Dashboard Generator is an open-source dashboard generator based on Next.js and ChatGPT. The tool allows users to feed existing tables or datasets, and from there, it understands the intent of the table and generates a visual representation of the data.
 
-First, run the development server:
+Internally, AI Dashboard Generator uses a custom CSV dataset and ChatGPT with a prompt model contextualized around JSON responses to give simple and useful visualizations. You can filter the parameters and export the dataviz directly into javascript code.
 
-```bash
-npm run dev
-# or
-yarn dev
+## Installation
+
+To get started with the AI Dashboard Generator, you'll need to have Node.js and npm installed on your machine. Once you've done that, you can install the dependencies by running the following command in your terminal:
+
+```
+> npm install
```

Comment:
Maybe we should add here or in another sections the instructions on how to run the code after it's installed?